### PR TITLE
Other geico

### PR DESCRIPTION
**Pull request by ld3300**
Add Geico to Other section

### CODE REVIEW

**Comment by mxxcon** [comment]
Hmm..would be considered `other` or maybe `utilities`?

**Comment by ld3300** [comment]
I'm open to suggestions.  Kind of felt between categories.

**Comment by jamcat22** [comment]
I say `finance`

**Comment by mxxcon** [comment]
Actually @jamcat22 and I talked and think that `finance` would be an even more suitable category.

File: _data/finance.yml [hunk 1]
```diff
@@ -17,6 +17,12 @@ websites:
       twitter: freshbooks
       tfa: No
 
+    - name: Geico
+      url: https://www.geico.com/
+      img: geico.png
+      twitter: geico
```
Comment (Carlgo11) [inline _data/finance.yml]: Looks like they answer questions on `GEICO_Service`.
Change the Twitter handle please

**Pull request merged**